cmd/simulate.go

```
@@ -0,0 +1,49 @@
+	"github.com/ganimtron-10/TriFS/internal/client"
+	"github.com/ganimtron-10/TriFS/internal/master"
+	"github.com/ganimtron-10/TriFS/internal/transport"
+	"github.com/ganimtron-10/TriFS/internal/worker"
+)
+
+func StartMaster() {
+	coreMaster := master.CreateMaster()
+
+	masterService := master.CreateMasterService(coreMaster)
+
+	transport.StartRpcServer(fmt.Sprintf(":%d", coreMaster.Port), masterService)
```

golangci-lint (1.64.8)

Error return value of `transport.StartRpcServer` is not checked

(errcheck)